test/unit/bloom.cc

```diff
@@ -1,7 +1,7 @@
-#include "salad/bloom.h"
 #include <unordered_set>
 #include <vector>
 #include <iostream>
+#include "salad/bloom.h"
```

Comment:
We don't have a rule stating that system headers must be included before tt headers. Moreover, we typically include the tested header first in a unit test. So the fix is fragile - someone may reintroduce the issue by adding another source file or "fixing" this one. Please fix the alignas definition in trivia/util.h instead.

According to https://en.cppreference.com/w/cpp/language/alignas, `stdalign.h` defines `__alignas_is_defined` in C++. Maybe, we should include it into trivia/util.h before checking `__alignas_is_defined`?

Reply:
I changed to unconditional include of "#include <stdalign.h>" for C++.